tests/engines/registries/acceptance/overview/overview-test.ts

```diff
@@ -156,7 +156,7 @@ module('Registries | Acceptance | overview.overview', hooks => {
         reg.update({ currentUserPermissions: [Permission.Write, Permission.Read] });
         await visit(`/${reg.id}/`);
 
-        assert.dom('[data-test-tags-widget-tag-input="edit"] input').isVisible();
+        assert.dom('[data-test-tags-widget-tag-input="edit"] input').isNotVisible();
```

Comment:
This doesn't seem correct. Why can't write users edit tags anymore?

Reply:
Taking a look.

Reply:
This has now been updated with a click for its edit button on the overview page.